strawberry/contrib/django/converter.py

```diff
@@ -0,0 +1,67 @@
+from functools import singledispatch
+from typing import Optional
+
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+
+import strawberry
+
+
+@singledispatch
+def convert_django_field(field):
+    raise ImproperlyConfigured(
```

Comment:
is this the right exception? we could change this to a custom one, maybe in another pr :)

Reply:
@patrick91 I think this is fine for now. It's the same error [graphene-django](https://github.com/graphql-python/graphene-django/blob/865c9535b969816a818e841c7ca219c992cfbf68/graphene_django/forms/converter.py#L15) throws when it can't properly convert a field.